handbook/growth.md

```diff
@@ -86,3 +86,46 @@ The following [Slack channels are maintained](https://fleetdm.com/handbook/compa
 
 
 <meta name="maintainedBy" value="timmy-k">
+
+## Growth Rituals
```

Comment:
Fleet uses sentence case capitalization in headings:
https://fleetdm.com/handbook/brand#how-to-write-headings-subheadings